doc/05-service-monitoring.md

```diff
@@ -646,22 +646,46 @@ Icinga sets `LC_NUMERIC=C` to enforce this locale on plugin execution.
 
 ##### Unit of Measurement (UOM) <a id="service-monitoring-plugin-api-performance-data-metrics-uom"></a>
 
-Unit     | Description
----------|---------------------------------
-None     | Integer or floating point number for any type (processes, users, etc.).
-`s`      | Seconds, can be `s`, `ms`, `us`.
-`%`      | Percentage.
-`B`      | Bytes, can be `KB`, `MB`, `GB`, `TB`. Lowercase is also possible.
-`c`      | A continuous counter (e.g. interface traffic counters).
-
-Icinga metric writers normalize these values to the lowest common base, e.g. seconds and bytes.
-Bad plugins change the UOM for different sizing, e.g. returning the disk usage in MB and later GB
-for the same performance data label. This is to ensure that graphs always look the same.
-
 ```
 'rta'=12.445000ms 'pl'=0%
 ```
 
+Icinga interprets the plugins' UoMs as follows:
+
+* If the UoM is "c", the value is a continuous counter (e.g. interface traffic counters).
+* Otherwise if the UoM is listed in the table below (case-insensitive if possible),
+  Icinga normalizes the value (and warn, crit, min, max) to the respective common base.
+  That common base is also used by the metric writers (if any).
+
+Common base        | UoMs
+-------------------|---------------------------------------
+bytes              | B, KB, MB, ..., YB, KiB, MiB, ..., YiB
+bits               | b, kb, mb, ..., yb, kib, mib, ..., yib
+packets            | packets
+seconds            | ns, us, ms, s, m, h, d
+percent            | %
+amperes            | nA, uA, mA, A, kA, MA, GA, ..., YA
+ohms               | nO, uO, mO, O, kO, MO, GO, ..., YO
+volts              | nV, uV, mV, V, kV, MV, GV, ..., YV
+watts              | nW, uW, mW, W, kW, MW, GW, ..., YW
+ampere-seconds     | nAs, uAs, mAs, As, kAs, MAs, GAs, ..., YAs, all of these also for Am and Ah
+watt-hours         | nWh, uWh, mWh, Wh, kWh, MWh, GWh, ..., YWh, all of these also for Wm and Ws
+lumens             | lm
+decibel-milliwatts | dBm
+grams              | ng, ug, mg, g, kg, t
+degrees-celsius    | C
+degrees-fahrenheit | F
+degrees-kelvin     | K
+liters             | ml, l, hl
+
+Bad plugins change the UOM for different sizing, e.g. returning the disk usage in MB and later GB
```

Comment:
- I'd not call a plugin `bad`, that's a bit to judgmental for a documentation in my opinion. Maybe just use `some`?
- UoM not UOM (typo)